python/ray/train/v2/tests/test_logging.py

```diff
@@ -100,14 +100,23 @@ def test_controller_sys_logged_to_file(controller_logging, context):
     train_logger = logging.getLogger("ray.train.spam")
     train_logger.info("ham")
 
-    log_path = os.path.join(
-        get_log_directory(), f"ray-train-sys-controller-{worker_id}.log"
-    )
-    with open(log_path, encoding="utf-8") as file:
-        log_contents = file.read()
+    log_contents = get_file_contents(f"ray-train-sys-controller-{worker_id}.log")
     assert "ham" in log_contents
 
 
+@pytest.mark.parametrize("context", [get_dummy_run_context()])
+def test_controller_sys_not_logged_to_file(controller_logging, context):
```

Comment:
can you make this a `@pytest.mark.parametrize("configure_logging", [True, False])`?

And also, can you do a driveby change to convert the current `"context"` parameter to just a regular fixture since it's always a single value.

Reply:
Converted `context` - lmk if this is what you had in mind since I'm new to pytest. 

I initially tried `can you make this a @pytest.mark.parametrize("configure_logging", [True, False])` but decided against it because the branching logic (toggling whether we call `configure_controller_logger` and then assert contents vs assert raises) is even more lines of code. Lmk what you think.

Reply:
I think I prefer these 2 tests to be packaged in the same method even if it's more lines of code, to make it clear that the 2nd case is the omission of the call to configure_x_logging. But I'm not going to block merge on this nit